CodeAssignment_MVVM_Combine/Sources/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,309 @@
+//
+//  DetailViewController.swift
+//  CodeAssignment_MVVM_Combine
+//
+//  Created by gyusoku.i on 2024/05/16.
+//
+import UIKit
+import SnapKit
+import SDWebImage
+
+private enum Const {
+    /// layout設定で使うleftPadding
+    static let leftPadding: CGFloat = 20
+    /// layout設定で使うRightPadding
+    static let rightPadding: CGFloat = 20
+    /// DetailViewControllerで表すLanguageColorViewのHeight
+    static let colorViewHeight: CGFloat = 20
+}
+
+final class DetailViewController: UIViewController {
+    /** Vertical方向のScrollView
+    - ScrollViewで、backgroundCardViewをScroll可能にする
+     */
+    private lazy var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.alwaysBounceVertical = true
+        scrollView.showsVerticalScrollIndicator = true
+        return scrollView
+    }()
+    
+    /// リポジトリのデータを入れるbackgroundView
+    private lazy var backgroundCardView: UIView = {
+        let view = UIView()
+        view.clipsToBounds = true
+        view.layer.cornerRadius = 15
+        view.backgroundColor = .white
+        return view
+    }()
+    
+    /// リポジトリ名を表示するlabel
+    private lazy var repositoryNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 20, weight: .semibold)
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    /// Descriptionを表示するlabel
+    private lazy var descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 18, weight: .regular)
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    /// UserImageViewを入れるためのContentView
+    private lazy var userImageContentView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    /// Userのプロフィール画像
+    private lazy var userImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.image = UIImage(systemName: "person.circle")?.withTintColor(.systemMint, renderingMode: .alwaysOriginal)
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = 8
+        return imageView
+    }()
+    
+    /// User名を表示するためのlabel
+    private lazy var userNameLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.font = .systemFont(ofSize: 22, weight: .bold)
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    /// 当該リポジトリのStarの数も一緒に表示するStarボタン
+    private lazy var starButton: UIButton = {
+        var config = UIButton.Configuration.filled()
+        let image = UIImage(systemName: "star")?.withTintColor(.systemGray3, renderingMode: .alwaysOriginal)
+        if let image {
+            config.image = image.withConfiguration(UIImage.SymbolConfiguration(pointSize: 25, weight: .regular))
+        }
```

Comment:
ここってそもそもconfig.imageが受け取るのは `Image?` 型だから、if letをやる必要ないんじゃないかな？
```suggestion
        config.image = UIImage(systemName: "star")?.withTintColor(.systemGray3, renderingMode: .alwaysOriginal).withConfiguration(UIImage.SymbolConfiguration(pointSize: 25, weight: .regular))
```

Reply:
[a798e26](https://github.com/KyusokLee/CodeAssignment_MVVM_Combine/pull/5/commits/a798e26e9c0db810c0885d866e2aa90f9a8ad337)にてコード修正しました！ご確認お願いします！